billy_metadata/oh.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import lxml.html
```

Comment:
Some of these imports shouldn't be needed after moving or deleting functions below.